giotto/images/preprocessing.py

```diff
@@ -0,0 +1,113 @@
+"""Image preprocessing."""
+# License: Apache 2.0
+
+import numbers
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from joblib import Parallel, delayed, effective_n_jobs
+from sklearn.utils import gen_even_slices
+from sklearn.utils.validation import check_is_fitted, check_array
+from ..utils.validation import validate_params
+
+
+class Binarizer(BaseEstimator, TransformerMixin):
```

Comment:
Have you seen @ulupo 's wrapper `@adapt_fit_transform_docs` in `giotto/utils/_docs.py`? Perhaps you can include it here given the output of the class is "exotic"